src/cli/index.ts

```diff
@@ -56,7 +56,8 @@ export async function processArgv(): Promise<void> {
   try {
     await cli(process.argv.slice(2))
     process.exit(0)
-  } catch (err) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } catch (err: any) {
```

Comment:
is there any harm in using `unknown` here and then adding a check for the type of `err` before doing what this block does?

Reply:
same below

Reply:
Ts is not happy if we let it auto infer as `unknown` because we can’t access a property from `unknown`, e.g. the error is “property message doesn’t exist in type unknown”.

Maybe we still can specify the type of expected error.

Reply:
Would something [like this](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABAUwE6rqgEgQwM4CyyeeOA5sgBRqoBci4A1mHAO5gCU9NiMeiAbwC2xUhXp4oqGGDIBfQQChEK3sESUoATwAOyOOp4BCALwnEAcjgAjAFbJoFxAB9nKdIjPmwIADa+ORFRkKBBUJGAcXzxkAG5FZVVg0PDLERJyZCcZd1R4hQTQSFgERChiKEpgegAxcGh4TiVVMtQtZpbVYEoOeJaFCBwoCAALDRp6JhZ2QIFEzrVx9ExcQlFM6nQOWfmF1QgEPDhfZAA6XzgyTdRT9LFkXt3+3blFOSA) help?

Reply:
nice I like that, thanks !!